.github/workflows/extra-docker-images.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Build and push xmpp-prosody-chat-server
         uses: docker/build-push-action@v5
         with:
-          context: "{{defaultContext}}:docker/full-tls-mutinynet/prosody"
+          context: "{{defaultContext}}:docker/0.2.1/full-tls-mutinynet/prosody"
```

Comment:
Should we ... get it from a current tag or something?

Reply:
We had a `latest` dir, but that was removed in https://github.com/fedimint/fedimint/commit/0f3a65268c06e9a592d87a6c2e8c145a53139735

@Kodylow (tagging since you have context)

Do you have a pattern you'd like to use so we can reference Dockerfiles similar to a current tag?

Reply:
I'm happy doing latest if those are stable, I tagged to 0.2.1 as the stable release. If we do latest will it pull 0.2.2 or will it be pulling master updates?

Reply:
Since we're statically defining the path in the repo, we shouldn't be pulling (please correct me if I'm misinterpreting).

For context, this is the build and push job in CI, which is failing since the previous directory structure was refactored. We just need to point to the correct path.

```
failed to read dockerfile: open docker/mutinynet-bitcoind-docker/Dockerfile: no such file or directory
```

The job fails at the previous step for bitcoind, but we'll also fail for the prosody path.

Reply:
What I meant was: ... do only on tag builds and use some `${{ something.tag-name }}` kind of thing to get the version automaticaly?

Reply:
I believe we're already getting the version automatically from the `docker/metadata-action@v5` steps.

Just to make sure I understand, are you suggesting we remove `branches` from the workflow and only run the job on tag events?

https://github.com/fedimint/fedimint/blob/master/.github/workflows/extra-docker-images.yml#L5-L7

Also, we still need to give the job a path to the Dockeriles/contexts. For now, the common place appears to be `fedimint/docker/0.2.1/...`. I'm curious if there's a pattern where the common artifacts (i.e. bitcoind and prosody) should live that isn't 0.2.1.

Reply:
I opened #4262 to discuss